clusters/app.ci/cert-manager/rhcos-redirector_ingress.yaml

```diff
@@ -10,6 +10,8 @@ spec:
     - secretName: rhcos-redirector-tls
       hosts:
         - rhcos-redirector.ci.openshift.org
+        - releases-rhcos.svc.ci.openshift.org
```

Comment:
I think we can actually drop this one, it's not important.

Reply:
Dropped.